.github/workflows/upstream.yml

```diff
@@ -7,108 +6,137 @@ on:
 
 concurrency:
   group: upstream-${{ github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: false
```

Comment:
Why is this false? If you push a new commit, you probably don't care about the old one finishing, regardless of the fact this is a development build?